services/web3/useGetMinimumClaimDeadline.tsx

```diff
@@ -4,13 +4,11 @@ import { psyClaimsMainnet, psyClaimsSepolia } from "@/constants/contracts";
 import { env } from "process";
 
 export const useGetMinimumClaimDeadline = () => {
-    const { data, isSuccess, refetch, error } = useReadContract({
-        address: env.NEXT_PUBLIC_IS_MAINNET
-        ? psyClaimsMainnet
-        : psyClaimsSepolia,
-      abi: psyClaimsAbi,
-      functionName: "minimumClaimDeadline",
-    });
-  
-    return { minimumClaimDeadline: data as boolean, isSuccess, refetch, error };
-  };
+  const { data, isSuccess, refetch, error } = useReadContract({
+    address: env.NEXT_PUBLIC_IS_MAINNET ? psyClaimsMainnet : psyClaimsSepolia,
+    abi: psyClaimsAbi,
+    functionName: "minimumClaimDeadline"
+  });
+
+  return { minimumClaimDeadline: data as bigint, isSuccess, refetch, error };
```

Comment:
the data from this will give us the timestamp we need to calculate the minimum date